### ISSUE TITLE
Submit job sometime get 500 internal server error

### ISSUE THREAD

**Issue by helinwang** [PaddlePaddle/cloud#336]
```
➜  demo git:(develop) ✗ paddlecloud submit -jobname b -cpu 1 -gpu 0 -parallelism 1 -faulttolerant=true -entry "python train_ft.py" fit_a_line
Uploading ... fit_a_line/train.py /pfs/dlnel/home/helinwang@baidu.com/jobs/b/train.py
Uploading ... fit_a_line/train_ft.py /pfs/dlnel/home/helinwang@baidu.com/jobs/b/train_ft.py
error submiting job: server error: 500 Internal Server Error
```

```
➜  demo git:(develop) ✗ paddlecloud kill c
➜  demo git:(develop) ✗ paddlecloud submit -jobname a -cpu 1 -gpu 0 -parallelism 1 -faulttolerant=false -entry "python train.py" fit_a_line
Uploading ... fit_a_line/train.py /pfs/dlnel/home/helinwang@baidu.com/jobs/a/train.py
Uploading ... fit_a_line/train_ft.py /pfs/dlnel/home/helinwang@baidu.com/jobs/a/train_ft.py
error submiting job: server error: 500 Internal Server Error
```

```
➜  demo git:(develop) ✗ paddlecloud kill a
➜  demo git:(develop) ✗ paddlecloud submit -jobname a -cpu 1 -gpu 0 -parallelism 1 -faulttolerant=false -entry "python train.py" fit_a_line
Uploading ... fit_a_line/train.py /pfs/dlnel/home/helinwang@baidu.com/jobs/a/train.py
Uploading ... fit_a_line/train_ft.py /pfs/dlnel/home/helinwang@baidu.com/jobs/a/train_ft.py
error submiting job: server error: 500 Internal Server Error
```

The 500 internal server error will persist for few seconds / minutes, and then everything becomes fine again.

**Comment by Yancey1989** [comment]
Sorry, it seems like a bug, and fixed PR: #333

**Comment by helinwang** [comment]
Thanks!!! Now I am using without any problem.
